<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1239</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="Project_Path_Hbox">
   <widget class="QPushButton" name="Generate_Project_Files_Btn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>430</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Setup Project Files</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>1061</width>
      <height>260</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,45,1">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="MainLayout">
       <property name="spacing">
        <number>50</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="ProjectPath_Hbox" stretch="0,30,11">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="Project_Path_Label">
           <property name="text">
            <string>Project Path      </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Project_Path_Txt"/>
         </item>
         <item>
          <widget class="QPushButton" name="Project_Path_Browse_Btn">
           <property name="text">
            <string>Set Project Path</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="UE_Source_Path_Hbox" stretch="0,30,11">
         <item>
          <widget class="QLabel" name="UE_Source_Path_Label">
           <property name="text">
            <string>UE Source Path</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="UE_Source_Path_Txt"/>
         </item>
         <item>
          <widget class="QPushButton" name="UE_Source_Path_Browse_Btn">
           <property name="text">
            <string> Engine Path</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="Project_Name_HBOX" stretch="0,30,11">
         <item>
          <widget class="QLabel" name="ProjectName">
           <property name="text">
            <string>Project Name   </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Project_Name_Txt">
           <property name="inputMethodHints">
            <set>Qt::ImhLatinOnly</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Project_Name_Error">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>881</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1239</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
